core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -128,11 +152,36 @@ private[ui] class ExecutorsPage(
       <td sorttable_customkey={diskUsed.toString}>
         {Utils.bytesToString(diskUsed)}
       </td>
-      <td>{info.activeTasks}</td>
-      <td>{info.failedTasks}</td>
-      <td>{info.completedTasks}</td>
+      <td style={
+        if (info.activeTasks > 0) {
+          "background:hsla(120, 100%, 25%, " + activeTasksAlpha + ");color:white"
+        } else {
+          ""
+        }
+      }>{info.activeTasks}</td>
+      <td style={
+        if (info.failedTasks > 0) {
+          "background:hsla(0, 100%, 50%, " + failedTasksAlpha + ");color:white"
```

Comment:
That is actually how HSLa is designed and was it seemed to be an area on contention when it was first defined.